tools/extract-spec-data.js

```diff
@@ -0,0 +1,166 @@
+/*******************************************************************************
+Helper script that parses data files and fetches additional information from
+the W3C API.
+
+To parse files:
+node tools/extract-spec-data.js dta/3dcamera.json data/webvtt.json
+*******************************************************************************/
+
+const fetch = require('fetch-filecache-for-crawling');
+const path = require('path');
+const DOMParser = require('xmldom').DOMParser;
+const xpath = require('xpath');
+
+const namespaces = {
+  'c': 'http://www.w3.org/2000/10/swap/pim/contact#',
+  'o': 'http://www.w3.org/2001/04/roadmap/org#',
+  'd': 'http://www.w3.org/2000/10/swap/pim/doc#',
+  'rdf': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#',
+  'rec': 'http://www.w3.org/2001/02pd/rec54#',
+  'dc': 'http://purl.org/dc/elements/1.1/'
+};
+const xselect = xpath.useNamespaces(namespaces);
+
+const maturities = ['LastCall', 'WD', 'CR', 'PR', 'REC', 'Retired', 'NOTE'];
+
+
+/**
+ * Wrapper around the "require" function to require files relative to the
+ * current working directory (CWD), instead of relative to the current JS
+ * file.
+ *
+ * This is typically needed to be able to use "require" to load JSON config
+ * files provided as command-line arguments.
+ *
+ * @function
+ * @param {String} filename The path to the file to require
+ * @return {Object} The result of requiring the file relative to the current
+ *   working directory.
+ */
+function requireFromWorkingDirectory(filename) {
```

Comment:
Maybe we can extract utility functions like this to a new file in the future.

Since we do not have many utility functions currently, this looks good to me now.